pyvista/core/filters/data_set.py

```diff
@@ -1533,11 +1533,11 @@ def threshold(
         if scalars is None:
             set_default_active_scalars(self)  # type: ignore[arg-type]
             _, scalars = self.active_scalars_info  # type: ignore[attr-defined]
-        arr = get_array(self, scalars, preference=preference, err=False)
+        arr = get_array(self, scalars, preference=preference, err=False)  # type: ignore[arg-type]
```

Comment:
These new ignores are similar to the other `# type: ignore[attr-defined]` ignores used everywhere in DataSetFilters. DataSet is expected for `self` but `DataSetFilters` is given. #6870 should fix both the `# type: ignore[attr-defined]` and the new `# type: ignore[arg-type]` errors.